core/domain/topic_services.py

```diff
@@ -229,6 +287,13 @@ def apply_change_list(topic_id, change_list):
                 elif (change.property_name ==
                       topic_domain.TOPIC_PROPERTY_LANGUAGE_CODE):
                     topic.update_language_code(change.new_value)
+            elif change.cmd == topic_domain.CMD_UPDATE_SUBTOPIC_PROPERTY:
+                if (change.property_name ==
+                        topic_domain.SUBTOPIC_PROPERTY_TITLE):
+                    topic.update_subtopic_title(change.id, change.new_value)
+                elif (change.property_name ==
```

Comment:
might want to reframe as a move perhaps? Ensures that after each cmd operation, stuff is still internally consistent.

Reply:
Done.